com.aobuchow.themes.spectrum.preferences/src/com/aobuchow/themes/spectrum/preferences/SpectrumPreferencePage.java

```diff
@@ -0,0 +1,112 @@
+	private Color accentColor;
+	private Color baseColor;
+	private Color backgroundColor;
+
+	@Override
+	public void init(IWorkbench workbench) {
+		colorRegistry = PlatformUI.getWorkbench().getThemeManager().getCurrentTheme().getColorRegistry();
+		accentColor = colorRegistry.get("com.aobuchow.themes.spectrum.ACCENT_COLOR");
+		baseColor = colorRegistry.get("com.aobuchow.themes.spectrum.BASE_COLOR");
+		backgroundColor = colorRegistry.get("com.aobuchow.themes.spectrum.BACKGROUND_COLOR");
+		
+		try {
+			browser = PlatformUI.getWorkbench().getBrowserSupport().createBrowser("GitHub");
+		} catch (PartInitException e) {
+			// Internal browser shouldn't cause an exception
+			Activator.getDefault().getLog().log(new Status(ERROR, getClass(), e.getMessage()));
+		}
+
+		try {
+			browser = PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser();
+		} catch (PartInitException e) {
+			Activator.getDefault().getLog().log(new Status(INFORMATION, getClass(), e.getMessage()));
+		}
+		try {
+			issuesURL = new URL("https://github.com/AObuchow/Eclipse-Spectrum-Theme/issues");
+			repoURL = new URL("https://github.com/AObuchow/Eclipse-Spectrum-Theme");
+		} catch (MalformedURLException e) {
+			Activator.getDefault().getLog().log(new Status(ERROR, getClass(), e.getMessage()));
+		}
+	}
+
+	@Override
+	protected Control createContents(Composite parent) {
+		Composite composite = new Composite(parent, SWT.NONE);
+		GridLayout layout = new GridLayout(1, true);
+		layout.marginWidth = 0;
+		layout.marginHeight = 0;
+		composite.setLayout(layout);
+		
+		Group customizeGroup = new Group(composite, SWT.SHADOW_ETCHED_IN);
+		customizeGroup.setText("Customization:");
+		customizeGroup.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
+		customizeGroup.setLayout(layout);
+		Text colorScheme = new Text(customizeGroup, SWT.LEAD | SWT.BORDER | SWT.MULTI | SWT.READ_ONLY);
+		colorScheme.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
+		String accentColorHex = String.format(rgbToHex, accentColor.getRed(), accentColor.getGreen(), accentColor.getBlue());
+		String baseColorHex = String.format(rgbToHex, baseColor.getRed(), baseColor.getGreen(), baseColor.getBlue());
+		String backgroundColorHex = String.format(rgbToHex, backgroundColor.getRed(), backgroundColor.getGreen(), backgroundColor.getBlue());
+		String currentColorScheme = "ModernDark {\n" + 
+				"    --background-color: " + backgroundColorHex + ";\n" + 
+				"    --base-color: " + baseColorHex + ";\n" + 
+				"    --accent-color: " + accentColorHex + ";\n" + 
+				"}";
+		colorScheme.setText(currentColorScheme);
+
+		Group communityGroup = new Group(composite, SWT.SHADOW_ETCHED_IN);
+		communityGroup.setText("Get involved:");
+		communityGroup.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
+		communityGroup.setLayout(layout);
+		Link repoLink = new Link(communityGroup, SWT.BORDER | SWT.BOLD);
+		repoLink.setText("Project repository <a>link.</a>");
```

Comment:
The differences in the way the links are presented to the reader is a bit awkward.